apps/cowswap-frontend/index.html

```diff
@@ -25,17 +25,17 @@
       property="og:description"
       content="CoW Swap finds the lowest prices from all decentralized exchanges and DEX aggregators & saves you more with p2p trading and protection from MEV"
     />
-<!--    TODO: change after halloween-->
-<!--    <meta property="og:image" content="https://swap.cow.fi/images/og-meta-cowswap.png?v=2" />-->
-    <meta property="og:image" content="https://swap.cow.fi/images/og-meta-halloween.png?v=1" />
+
+    <meta property="og:image" content="https://swap.cow.fi/images/og-meta-cowswap.png?v=2" />
+<!--    <meta property="og:image" content="https://swap.cow.fi/images/og-meta-halloween.png?v=1" />-->
```

Comment:
Remove the commented out img?

Reply:
Left it for the next halloween to not forget

Reply:
Yes, I also don't love these dead code things. If its files in the project bothers me less. 
At this point, i think we have 3 PRs that cleaned up halloween, so it will be hard next time to find them to revert. 

Maybe we can clean the code and leave a note for our future selves, but we might forget. Or we can use the calendar to send us a reminder in October 2024 :)

Reply:
I agree 😄 Will delete it in develop